packages/base-styles/_variables.scss

```diff
@@ -45,7 +45,7 @@ $grid-unit-80: 8 * $grid-unit;		// 64px
  */
 
 $icon-size: 24px;
-$button-size: 36px;
+$button-size: 32px;
```

Comment:
This change is going to have an impact on a lot of places, I'm going to list them here so we can test them:

 - Block directory download buttons
 - Block switcher button in the block toolbar (when there are transforms and also when it's disabled)
 - The quick inserter "browse all" button
 - The block title button in navigation mode
 - List view move and menu buttons
 - Block icons in the block inspector when doing a multi block selection
 - URL input suggestions and buttons in the link control modal (when adding links)
 - Classic block editor height (we compute the height by subtracting the button height)
 - image block aspect ratio button
 - Block navigation placeholder min height, navigation block placeholder icons, indicators and fields
 - Table block placeholder inputs
 - Some mobile/native styles in social link blocks.
 - The "/" menu suggestions
 - DropdownMenu component menus (there's a min height for the toggle using this variable)
 - The guides footer height (welcome guides ...)
 - The modal header icon sizes
 - Min height of panel rows (I'm guessing that's a wrapper of each "control" in the block inspectors)
 - Toolbars with dividers (block toolbar, header toolbars)
 - The "document actions" (center of the top editor toolbar) in the template mode of the post editor.
 - Fullscreen close button in post editor (I'm not sure this component is used anymore)
 - Document overview panel (close button)
 - Inserter height in mobile (do we leave enough room for the close button)
 - Line height of the code editor (both site and post editors)
 - Check the site editor canvas header in mobile
 - Site editor header obviously
 - Site editor document actions (thing in the center of the top toolbar)
 - Minimum width of the "actions" column in the template and template parts browse all pages
 - List view and inserter panel heights (I'm guessing in all editors in mobile viewports: post editor, site editor and widget editor)
 - Widget editor top toolbar buttons
 - Height of post publish panel input (I'm guessing the one showing the URL)

Reply:
Yeah this change should probably be avoided — since the `32px` button is the one associated to `size="compact"`, we should use the `$button-size-compact` variable when necessary (declared just a few lines below)